VS 21 February/VS 21 February/Task 6.2 doctor/Doctor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+namespace VS_21_February
+{
+	public class Doctor
+	{
+        public string name;
+        public double experience;
+        public int qualificationCategory;
+
+        public Doctor(string name, double experience, int qualificationCategory)
```

Comment:
cosmetic

Reply:
I'am a little bit confused. What exactly should I fix in this part of the code?

Reply:
fixed